index.html

```diff
@@ -0,0 +1,67 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+    <header class="container">
+
+        <nav class="toolbar">
+            <div class="logo"><a class="logo-txt" href="#">My Logo</a></div>
+            <div class="menu-image"><img src="./images/ic_menu.jpg" alt="menu"></div>
+        </nav>
+
+        <div class="circle">
+            <br id="rec">
+
+        </div>
+
+
+        <section class="headline">
+            <div class="intro">
+                <h1>i'm Adan Glad to see you!</h1>
+            </div>
+            <div class="sumarry">
+                <p class="about-me">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                    to contact me.</p>
+            </div>
```

Comment:
- You did a great job here but make sure your design mutch to the Figma design as required by the previous reviewer. :+1: :grinning:  :grinning:  :grinning: 
You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).